pkg/ami/static_resolver_ami_generate.go

```diff
@@ -0,0 +1,80 @@
+// +build ignore
+
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials/stscreds"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+
+	"github.com/weaveworks/eksctl/pkg/ami"
+	"github.com/weaveworks/eksctl/pkg/eks/api"
+
+	. "github.com/dave/jennifer/jen"
```

Comment:
Dot imports are pretty anti-pattern and I don't see any obvious reasons for it. The only general exception is in test code.

https://github.com/golang/go/wiki/CodeReviewComments#import-dot